### PR TITLE
Don't stop the query when stopping polling.

### PR DESCRIPTION
**Pull request by tmeasday**
You may still want to receive updates to the query from other (non-polling) sources.

@helfer does it make sense to add a test for this?

### CODE REVIEW

**Comment by helfer** [comment]
@tmeasday yeah, let's also add a test for this so we don't screw it up again.

**Comment by stubailo** [comment]
Ping: can we add a test here?

**Comment by tmeasday** [comment]
Done

**Comment by stubailo** [comment]
Thanks!

File: test/QueryManager.ts [hunk 1]
```diff
@@ -1921,17 +1921,75 @@ describe('QueryManager', () => {
...
 
-        if (handleCount === 2) {
+        if (handleCount === 1) {
           handle.stopPolling();
         }
       },
     });
 
     setTimeout(() => {
-      assert.equal(handleCount, 2);
```
Comment (tmeasday) [inline test/QueryManager.ts]: Note that this test wasn't really testing anything before. I mean it probably would thrown a network interface error I guess, but this seems like a more useful error to get.

**Comment by tmeasday** [comment]
Actually hold on, my test is bogus.

**Comment by tmeasday** [comment]
Ok, I fixed it.

**Pull request merged**